viewer/Viewer.h

```diff
@@ -111,23 +124,58 @@ namespace Enki
 		typedef QMapIterator<const std::type_info*, const std::type_info*> ManagedObjectsAliasesMapIterator;
 		ManagedObjectsAliasesMap managedObjectsAliases;
 		
+		typedef std::pair<QString, unsigned int> viewerMessage;
+		std::list<viewerMessage> messageList;
```

Comment:
no this is just an interface to add messages in the timed FIFO queue. But yes, maybe it's better to check in the functions to prevent message duplication